Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -88,7 +96,11 @@ below. This will ensure all pods are managed by Cilium.
       --set global.nodeinit.enabled=true \\
       --set nodeinit.reconfigureKubelet=true \\
       --set nodeinit.removeCbrBridge=true \\
-      --set global.cni.binPath=/home/kubernetes/bin
+      --set global.cni.binPath=/home/kubernetes/bin \\
+      --set global.k8s.requireIPv4PodCIDR=true \\
+      --set global.tunnel=disabled \\
+      --set global.endpointRoutes.enabled=true \\
+      --set global.native-routing-cidr=$NATIVE_CIDR
```

Comment:
Do we also need to do something explicit about disabling cilium-health here? I think we'll allocate an address to it that's not routable.

Reply:
The great news is that we don't have to. All IPs in the PodCIDR handed to us are fully-routable. The connectivity health checking is working as expected:
```
KVStore:                Ok   Disabled
Kubernetes:             Ok   1.15+ (v1.15.11-gke.9) [linux/amd64]
Kubernetes APIs:        ["CustomResourceDefinition", "cilium/v2::CiliumClusterwideNetworkPolicy", "cilium/v2::CiliumEndpoint", "cilium/v2::CiliumNetworkPolicy", "cilium/v2::CiliumNode", "core/v1::Endpoint", "core/v1::Namespace", "core/v1::Pods", "core/v1::Service", "networking.k8s.io/v1::NetworkPolicy"]
KubeProxyReplacement:   Probe   [NodePort (HYBRID, 30000-32767, XDP: NONE), HostPort, ExternalIPs, HostReachableServices (TCP, UDP)]
Cilium:                 Ok      OK
NodeMonitor:            Disabled
Cilium health daemon:   Ok
IPAM:                   IPv4: 7/255 allocated from 10.68.2.0/24,
Controller Status:      36/36 healthy
Proxy Status:           OK, ip 10.68.2.184, 3 redirects active on ports 10000-20000
Hubble:                 Disabled
Cluster health:         3/3 reachable   (2020-04-21T20:52:18Z)
```